app/MMB/src/scenes/tripServices/transportation/CurrentPositionButton.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import * as React from 'react';
+import { View, TouchableWithoutFeedback } from 'react-native';
+import { StyleSheet, TextIcon, Color } from '@kiwicom/mobile-shared';
+import { defaultTokens } from '@kiwicom/mobile-orbit';
+
+type Props = {|
+  +onPress: () => void,
+|};
+
+export default function CurrentPositionButton(props: Props) {
+  return (
+    <View style={styles.roundButton}>
+      <TouchableWithoutFeedback onPress={props.onPress}>
+        <View>
+          <TextIcon code="&quot;" style={styles.icon} />
+        </View>
+      </TouchableWithoutFeedback>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  roundButton: {
+    borderWidth: 1,
+    width: 40,
+    height: 40,
+    borderColor: defaultTokens.paletteWhite,
+    borderRadius: 40,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: defaultTokens.paletteWhite,
+    shadowOpacity: 0.3,
+    shadowRadius: 2,
+    shadowColor: Color.black,
```

Comment:
We are not supposed to use this color, you should use `paletteInkDark`

Reply:
Thanks!